closed/src/java.base/share/classes/sun/security/ec/NativeECUtil.java

```diff
@@ -158,4 +160,69 @@ static long encodeGroup(ECParameterSpec params) {
                                        n, n.length,
                                        h, h.length);
     }
+
+    /**
+     * Returns the native pointer to the provided EC public key.
```

Comment:
Suggest "for" would be better than "to" here; likewise for `getPrivateKeyNativePtr()`.

Reply:
Changed.